### PR TITLE
Explicitly adding the diirt repo.

### PR DESCRIPTION
**Pull request by shroffk**

### CODE REVIEW

**Comment by berryma4** [comment]
Why do we need this?  Doesn't the core/diirt feature bring everythin in?

**Comment by shroffk** [comment]
Yes,
That is how tycho should work...that is how things seemed to have worked in the past...

File: pom.xml [hunk 1]
```diff
@@ -19,254 +13,257 @@
...
+    <!-- VERSIONS -->
+    <tycho.version>1.0.0</tycho.version>
     <tycho-extras.version>${tycho.version}</tycho-extras.version>
-		<cs-studio.version>4.5</cs-studio.version>
-		<cs-studio-central.url>http://download.controlsystemstudio.org/product/${cs-studio.version}</cs-studio-central.url>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<jgit.dirtyWorkingTree>error</jgit.dirtyWorkingTree>
-		<baselineMode>fail</baselineMode>
+    <cs-studio.version>4.5</cs-studio.version>
+    <diirt.version>3.1.6</diirt.version>
```
Comment (berryma4) [inline pom.xml]: A lot of white space changes, just marking changes

File: pom.xml [hunk 2]
```diff
@@ -19,254 +13,257 @@
...
+      <layout>p2</layout>
+    </repository>
+    <repository>
+      <id>efx</id>
+      <url>${efx-site}</url>
+      <layout>p2</layout>
+    </repository>
+    <repository>
+      <id>diirt</id>
+      <url>http://diirt.download.controlsystemstudio.org/diirt/${diirt.version}</url>
```
Comment (berryma4) [inline pom.xml]: A lot of white space changes, just marking changes

**Pull request merged**